### PR TITLE
Clarify GOVERNANCE

### PR DESCRIPTION
**Pull request by JustinCappos**
This is meant to update the GOVERNANCE documentation in preparation for project graduation.

Borrowed some of the code of conduct text from the SPIFFE project.  Tried to make it as clear and explicit as possible throughout.

Feedback / changes are welcome.

Note that this PR *must* be merged / approved by the CB as it technically changes the governance rules.

Signed-off-by: Justin Cappos <justincappos@gmail.com>

### CODE REVIEW

**Comment by adityasaky** [comment]
The changes themselves look fine to me but I'll leave them to @SantiagoTorres. However, I wonder if these points belong outside of this repository now. We have multiple implementations and subprojects now. Perhaps we should move them all to in-toto/docs so they apply in-toto wide and point there in all other repos.

File: GOVERNANCE.md [hunk 1]
```diff
@@ -34,10 +53,22 @@ See [*Instructions for
...
+changes in the ITE process will be managed through new ITEs.
+
 ## Changes in maintainership
+Active contributors may be offered or request to be granted maintainer status.
+This requires approval from the CB and is done in consultation with the
+current maintainers.
 
-A contributor to the project must express interest in becoming a maintainer.
-The CB has the authority to add or remove maintainers.
+Maintainers may be moved to emertius status.  This is done at the discretion of
```
Comment (lukpueh) [inline GOVERNANCE.md]:
```suggestion
Maintainers may be moved to emeritus status.  This is done at the discretion of 
```

File: GOVERNANCE.md [hunk 2]
```diff
@@ -4,16 +4,35 @@ project consists of the in-toto
...
 reports](https://github.com/in-toto/in-toto#security-issues-and-bugs).
 A maintainer has the authority to approve or reject pull requests submitted by
-contributors.  The project's Consensus Builder (CB) is
-Santiago Torres-Arias <santiagotorres@purdue.edu, @santiagotorres>.
+contributors.  Any maintainer may also interact with the CNCF on behalf of the
+project.  
+
+The project's Consensus Builder (CB), who is also granted all maintainer 
+privileges and responsibilities, is Santiago Torres-Arias 
+<santiagotorres@purdue.edu, @santiagotorres>.
```
Comment (lukpueh) [inline GOVERNANCE.md]: User names are treated case insensitive in GitHub, but this is the way Santiago writes his it:

```suggestion
<santiagotorres@purdue.edu, @SantiagoTorres>.
```

File: GOVERNANCE.md [hunk 3]
```diff
@@ -34,10 +53,22 @@ See [*Instructions for
 Contributors*](https://github.com/in-toto/in-toto#instructions-for-contributors)
 for help.
 
+## Specification changes (ITEs) 
+The [in-toto enhancement (ITE)](https://github.com/in-toto/ITE) approval process which 
+changes the in-toto specification is listed in
```
Comment (lukpueh) [inline GOVERNANCE.md]:
```suggestion
The [in-toto enhancement (ITE)](https://github.com/in-toto/ITE) approval process, which 
changes the in-toto specification, is listed in 
```

**Review by lukpueh — COMMENTED** [review]
LGTM, modulo some minor nits.

**Comment by JustinCappos** [comment]
> LGTM, modulo some minor nits.

Accepted your suggested edits, thanks!

**Pull request merged**